components/remote-environment-broker/internal/config/config.go

```diff
@@ -28,10 +28,10 @@ type Config struct {
 	Logger                     logger.Config
 	Port                       int              `default:"8080"`
 	Storage                    []storage.Config `valid:"required"`
-	BrokerName                 string           `valid:"required"`
 	BrokerRelistDurationWindow time.Duration    `valid:"required"`
+	ClusterScopedBrokerName    string           `valid:"required"`
 	// Before adjusting ui-api-layer and tests, this flag by default should be set to true.
-	ClusterScopedBroker bool `default:"true"`
+	ClusterScopedBroker bool ` valid:"required"`
```

Comment:
our Configuration does not support properly defaults for boolean type. go-defaults will set ClusterScopedBroker to true if this value were equal zero value (it means false). Pointers are also not supported.